### ISSUE TITLE
Not compatible with OpenSSL 1.1

### ISSUE THREAD

**Issue by Arenlor** [TinyMUSH/TinyMUSH#5]
Running with `./configure ; openssl version` I get the following output at the end.
```
## -------------------- ##
## Checks for libraries ##
## -------------------- ##

checking for library containing crypt... -lcrypt
checking for library containing SSL_library_init... no
configure: error: in `/home/arenlor/TinyMUSH-master':
configure: error: could not find library openssl
See `config.log' for more details
OpenSSL 1.1.0h  27 Mar 2018
```

**Comment by moxventura** [comment]
From: https://www.openssl.org/docs/man1.1.0/man3/SSL_library_init.html

```The SSL_library_init() and OpenSSL_add_ssl_algorithms() functions were deprecated in OpenSSL 1.1.0 by OPENSSL_init_ssl().```

**Comment by EddyBeaupre** [comment]
Fixed in 3.3.0.10
